spec/controllers/people_controller_spec.rb

```diff
@@ -1,21 +1,23 @@
 require 'rails_helper'
 
 RSpec.describe PeopleController, type: :controller do
+  let(:user) { Fabricate(:user) }
 
   let(:valid_attributes) {
-    { first_name: "Jane", last_name: "Doe" }
+    { first_name: "Jane", last_name: "Doe", user_id: user.id }
   }
 
   let(:invalid_attributes) {
     { first_name: nil, last_name: nil }
   }
 
-  let(:valid_session) { {} }
+  let(:valid_session) { {user_id: user.id} }
 
   describe "GET #index" do
-    it "assigns all people as @people" do
-      person = Person.create! valid_attributes
-      get :index, {}, valid_session
+    it "assigns the current user's people" do
+      user = User.create
+      person = Person.create! valid_attributes.merge(user_id: user.id)
+      get :index, {}, { :user_id => user.id }
```

Comment:
Redundant curly braces around a hash parameter.<br>Use the new Ruby 1.9 hash syntax.